app/Resources/views/full/bundle_list.html.twig

```diff
@@ -6,14 +6,56 @@
             <div class="container">
                 <div class="row">
                     <div class="col-xs-12">
-                        {{ ez_render_field(content, 'description') }}
+                        <h1>{{ ez_render_field(content, 'name') }}</h1>
+                        <h2>{{ ez_render_field(content, 'description') }}</h2>
                     </div>
                 </div>
             </div>
         </div>
+        <div class="bundles-list-search">
+            <div class="container">
+                <div class="col-xs-12">
+                    {{ form_start(searchForm) }}
+                        <div class="form-group">
+                            {{ form_widget(searchForm.search, { 'id': 'bundles-list-search-query'} )}}
+                            <div class="input-group-addon">
+                                <button type="submit" class="btn bundles-list-search-button">
+                                    <i class="fa fa-search" aria-hidden="true"></i>
+                                </button>
+                            </div>
+                        </div>
+                    {{ form_end(searchForm) }}
+                </div>
+            </div>
+        </div>
         <div class="bundles-list-content">
             <div class="container">
-                <div class="col-xs-12 bundles-list-container">
+                <div class="hidden-xs hidden-sm col-md-2 bundle-list-settings">
+                    <h2>Categories</h2>
+                    <ul class="bundle-list-settings-list">
+                        <li class="active"><a href="#">All Bundles</a></li>
+                        <li><a href="#">Content Type</a></li>
+                        <li><a href="#">Field Type</a></li>
+                        <li><a href="#">Theme</a></li>
+                        <li><a href="#">Social</a></li>
+                        <li><a href="#">e-Commerce</a></li>
+                        <li><a href="#">Administration</a></li>
+                        <li><a href="#">Workflow</a></li>
+                    </ul>
+                </div>
+                <div class="col-xs-12 col-md-10 bundles-list-container">
+                    <div class="row">
+                        <div class="col-xs-12 col-md-9">
+                            <h2>All Bundles</h2>
+                        </div>
+                        <div class="col-xs-12 col-md-3 sort-container">
+                            {{ form_start(sortingForm) }}
+                                <div class="form-group {% if not sortingForm.order.vars.valid %}has-error{% endif %}">
+                                    {{  form_row(sortingForm.order) }}
```

Comment:
Yes, you are right. It is my mistake, I will change displayed value to `Popularity`, in search conditions I will keep `stars` because it depends on Github stars.